sources/org.osbuild.curl

```diff
@@ -163,6 +165,14 @@ class CurlSource(sources.SourceService):
                     "--location",
                     "--output", checksum,
                 ]
+                if url.endswith(".rpm"):
```

Comment:
How about just setting `User-Agent: osbuild (OSBuild; generic; Linux.{arch})` unconditionally? That would be more in line with the intention of the user agent  to identify the application/operating system/vendor [0] ?

[0] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
I'd include `osbuild.org` in our own UA so people can find us in case we ever end up on a naughty list somewhere.